lib/date_time_basics.dart

```diff
@@ -7,40 +7,52 @@ extension DateTimeBasics on DateTime {
   /// Returns true if [this] occurs strictly before [other], accounting for time
   /// zones.
   ///
-  /// Delegates to [DateTime.compareTo] and therefore obeys the contract of that
-  /// method.
-  bool operator <(DateTime other) => compareTo(other) < 0;
+  /// Alias for [DateTime.isBefore].
+  bool operator <(DateTime other) => this.isBefore(other);
```

Comment:
nit: I don't think you actually need the `this` on any of these.

Reply:
Yep, you're right. Removed now.